src/xdocs/google_style.xml

```diff
@@ -111,7 +111,7 @@
                             <td><a name="2.1"/><a href="#2.1"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s2.1-file-name">2.1 File name</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s2.1-file-name">2.1 File name</a>
                             </td>
```

Comment:
@romani `OneTopLevelClass` was not added a few lines down from here. (around 121).
Is it needed even though the class is mentioned elsewhere? We mention some checks multiple times.